git-and-github.md

```diff
@@ -140,6 +140,8 @@ Recommended Reading:
 
 Once your pull request has been *Approved* it may be merged at your discretion.  In most cases responsibility for merging is left to the person who opened the pull request, however for simple pull requests it is fine for anyone to merge.
 
+Please use `Squash and Merge` to keep the commit history clean on the main branch.
```

Comment:
I prefer to squash non-useful commits into useful ones with bullet descriptions and use `Rebase and merge`. Basically, if you clean up your commit history, `Rebase and merge`... if you did not and want to squash it at merge time, `Squash and merge`. Or some variation of that. Thoughts?

Reply:
I do both of those, support listing both options.

Reply:
Don't want to mandate any specific workflow, so agree with this. Maybe I'll call out that we typically avoid merge commits and do our best to keep the commit history clean and useful.

Reply:
that sgtm 👍🏼